### ISSUE TITLE
NVDA does not read aria-label on anchor tag in Chrome

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5332]
**Reported by mdwyer on 2015-09-09 13:53**
For anchor tags with text and an aria-label, NVDA only reads the anchor
 tag text, and not the aria-label in Google Chrome.  I am using NVDA 2015.3
 and Chrome 44.0.2403.157 m.  Note this behavior does not appear in Firefox
 40.0.3 or IE 11.

**Comment by jcsteh** [comment]
Chromium issue 527257 is fixed, so this now works as expected.
